### ISSUE TITLE
Kitava's Thirst trigger

### ISSUE THREAD

**Issue by creativity-x** [PathOfBuildingCommunity/PathOfBuilding#6545]
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

I am trying to trigger Kitava's Thirst using the surging thoughts harbinger sword, spending 100 mana by triggering the weapon spell socketed by Archmage support. The helmet proc trigger is the top line, but it does not trigger off the correct mechanic, and is not able to be modified as far as I can see, making the DPS calculation inaccurate.

### Describe the solution you'd like

Add a manual option for Kitava's Thirst Trigger rate

**Comment by Nightblade** [comment]
This sound more like a bug report, as it should work automatically AFAIK.  Could you provide a build code showing the problem please?

**Comment by creativity-x** [comment]
Yes sorry it is a bug report, I don't know what I'm doing on github, so I don't know if I can change that

Here is the build.... look at the left panel top line in blue text: "Kitava's Thirst Trigger"
https://pobb.in/drs-mQY7SeOZ

**Comment by Paliak** [comment]
Two issues:
* You don't have enough unreserved mana for archmage to increase the cost of Storm Cascade to 100.
* Nested triggers often have cyclic dependencies and those have a tendency to cause infinite recursion crashes. For most triggers it's currently not possible in POB to trigger a trigger with a triggered skill for this reason.

I have thought of implementing a sort of maximum recusion depth for triggers to allow for this kind of interactions but i'm not sure if or when i'll work on it.

Adding the ability to select a source skill from a filtered list would also be a nice feature but again. Not sure when or if.

**Comment by creativity-x** [comment]
Hi:
1.) Even with enough unreserved mana the issue with the interaction still exists. After getting rid of divine blessing, POB simply reports "No Kitava's Thirst Trigger".
2.) Completely makes sense. I assumed the lag was due to spellblade support, ha. Maybe incalculable in all honesty, its so many variables and triggers happening at once. I would say there's no need to work on anything. I will continue to just wing it and go by feel lol

**Comment by Nightblade** [comment]
@creativity-x Is this OK to close now?
